components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -41,11 +41,14 @@ This table shows snapshots of buckets in Storage.
 | `snapshot_date` (PK) | Date of the data snapshot | `2020-06-30` |
 | `stage` | Storage stage of the bucket | `in` |
 | `bucket` | Name of the bucket | `c-instagram` |
+| `bucket_display_name` | Name of the bucket | `instagram` |
 | `rows` | Number of rows in the bucket as of the date of the snapshot | `4714` |
 | `bytes` | Size of the bucket in bytes as of the date of the snapshot | `1870336` |
 | `sharing_type` | Type of bucket sharing (Data Catalog). <br> Possible values: `none` – not shared, <br> `target` – bucket linked from another project, <br>`source` / `org private` – bucket shared from this project, linkable only by organization members <br> `source` / `org public` – bucket shared from this project, linkable by any project members | `target` |
 | `shared_from_project_id` | Identifier of the source project if the bucket is linked (sharing_type = 'target') | `860_kbc-eu-central-1` |
 | `shared_from_bucket` | Identifier of the source bucket if the bucket is linked (sharing_type = 'target') | `in.c-keboola-ex-instagram-152387726` |
+| `is_external_schema` | Flag if the bucket contains data from external sources (`1`, `0`) | `1` |
```

Comment:
Tady chybi za "Flag" sloveso. U ostatních flags používáme `indicating`, tak bych se snažil být konzistentní a udělal to podle ostatních.

Reply:
@Matejkys upraveno i u ostatních, ať to máme jednotné.